src/styles/blocks/header.scss

```diff
@@ -0,0 +1,49 @@
+@use '../utils/mixins' as *;
+
+.header {
+  background-color: black;
+  background-repeat: no-repeat;
+  background-size: cover;
+  min-height: 100vh;
+  padding: 0 20px 80px;
+  box-sizing: border-box;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(2, 130px);
+  grid-template-rows: 1fr 1fr;
+  gap: 20px;
+
+  @include desktop {
+    background-image: url(/src/images/header-background-big.jpg);
+    grid-template-columns: repeat(12, 63px);
+    gap: 24px;
+  }
+
+  @include laptop {
+    background-image: url(/src/images/header-background-medium.jpg);
+  }
+
+  @media (max-width: 744px) {
+    background-image: url(/src/images/header-background-smaller.jpg);
+  }
```

Comment:
if you use mixins so you should to add all your breakpoinst as mixin, 
what is the different between laptop and tablet? on mock you have mobile, tablet and desktop resolutions